eoscostarica/eosio-spend-explorer
# Feat/claimed and unclaimed graphs

### Feat/claimed and unclaimed graphs

### What does this PR do?

- Resolve #99 
- Resolve #74
- Resolve #126 
- Resolve #127 

### Steps to test

1. Run project
2. Go to Income
3. You should see this:
![imagen](https://user-images.githubusercontent.com/49013556/191401908-d73e6ecf-9614-46fa-bf50-d756c98f65f0.png)


#### CheckList
